scripts/core/ScriptTestUtils.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Made this utility script that we can add various functions to to help out the command tests.